homeassistant/components/cookidoo/quality_scale.yaml

```diff
@@ -0,0 +1,80 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: No service actions implemented
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions:
+    status: exempt
+    comment: No service actions implemented
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions:
+    status: exempt
+    comment: No special external action required
+  entity-event-setup:
+    status: exempt
+    comment: No callbacks are implemented
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: done
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry: done
+
+  # Silver
+  config-entry-unloading: done
+  log-when-unavailable: done
+  entity-unavailable: done
+  action-exceptions:
+    status: exempt
+    comment: No service actions implemented
+  reauthentication-flow: done
+  parallel-updates: done
```

Comment:
this is todo, since you can define paralll updates for when you're writing out

Reply:
@zweckj Do I need to explicitly set `PARALLEL_UPDATES = 0` in the `todo.py` file?

Reply:
yep